packages/web/src/components/lineup/LineupProvider.tsx

```diff
@@ -68,20 +68,22 @@ const totalTileHeight = {
   playlist: 350
 }
 
+const innerHeight = typeof window !== 'undefined' ? window.innerHeight : 0
```

Comment:
legendary status to have fixed all cases where we access window. like react-native reloaded all over again